spec/supports/dependencies/factory_bot.rb

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 
 require 'factory_bot'
-FactoryBot.find_definitions
+RSpec.configure do |config|
```

Comment:
can we add this configuration to [here](https://github.com/nejdetkadir/devise-api/blob/main/spec/spec_helper.rb)?

Reply:
you can delete `spec/supports/dependencies/factory_bot.rb`file

Reply:
I believe that is correct, but the gem suggests adding the code to the factory_bot.rb file. @nejdetkadir

Reply:
Okey, thank you for all 🙏🏻